libfwupdplugin/fu-coswid-common.h

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (C) 2022 Richard Hughes <richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#pragma once
+
+#include <glib.h>
+
+#define FU_COSWID_GLOBAL_MAP_TAG_ID		       0
+#define FU_COSWID_GLOBAL_MAP_SOFTWARE_NAME	       1
```

Comment:
Could all these be an enum instead?

Reply:
Good idea, that also cleans up the code randomly too. Fixed.